ghost/core/core/server/data/schema/schema.js

```diff
@@ -497,6 +497,9 @@ module.exports = {
             }
         },
         attribution_url: {type: 'string', maxlength: 2000, nullable: true},
+        attribution_ref_source: {type: 'string', maxlength: 191, nullable: true},
+        attribution_ref_medium: {type: 'string', maxlength: 191, nullable: true},
+        attribution_ref_url: {type: 'string', maxlength: 2000, nullable: true},
```

Comment:
What's the difference between attribution_url and attribution_ref_url?

Reply:
@ErisDS `attribution_url` is the Ghost site URL from which member signed up or paid, for ex. a specific post or site home page. 
`attribution_ref_url`  captures the referrer's URL which led member to that particular post(or other) Ghost site URL, like Ghost Explore or Twitter.

Reply:
OK - I think we might want to rename these fields to `referrer_url`, `referrer_medium` and `referrer_source` and keep `attribution` to mean our internal post attribution?

Reply:
@allouis do you agree?

Reply:
Main reason for keeping the `attribution_` prefix was to have all attribution data(internal and external) grouped together for the event, but given the above confusion rename sounds good to me 👍

Reply:
@ErisDS I do 👍

Reply:
Also whilst we're here, before we have any "fun" the correct spelling is "referrer" with 2 R's... or 4 R's if you're a pedant 😂

Reply:
Haha all done to make sure we have the right "referrer" 😄